src/coreEnforcer.ts

```diff
@@ -485,7 +490,14 @@ export class CoreEnforcer {
       logPrint(reqStr);
     }
 
-    return res;
+    if(successReason){
+      response = { decision : res ,successRule : successReason };
```

Comment:
Maybe always return `[result, rule]` will be a better choice.

Reply:
Done, please check